packages/clerk-js/src/ui/components/OrganizationProfile/AddDomainPage.tsx

```diff
@@ -17,7 +17,7 @@ export const AddDomainPage = withCardStateProvider(() => {
   const nameField = useFormControl('name', '', {
     type: 'text',
     label: localizationKeys('formFieldLabel__organizationEmailDomain'),
-    placeholder: localizationKeys('formFieldInputPlaceholder__organizationName'),
+    placeholder: localizationKeys('formFieldInputPlaceholder__organizationEmailDomain'),
```

Comment:
@panteliselef , @nikosdouvlis isn't this a breaking change since a customer may have set their own localization for `formFieldInputPlaceholder__organizationName` ?

Reply:
I believe not as the Organization Domain feature is not yet released